docs/40_examples.md

```diff
@@ -69,3 +69,48 @@ You can customize the way your Project is deployed:
 ```sh
 miactl deploy development --no-semver --revision tags/v1.0.0
 ```
+
+## Deploy Project using a service account from a CD pipeline
+
+The commands are the same used above in the [Deploy Project](#deploy-project) section, but you need to use a _Service Account_ for that.
+If you don't know how to create a _Service Account_, read the [dedicated documentation](/docs/development_suite/identity-and-access-management/manage-service-accounts).
```

Comment:
this doc links should feature the full doc url; the aggregator will resolve the link when opening the PR